src/test/java/be/syntra/devshop/DevshopBack/controllers/ProductControllerTest.java

```diff
@@ -384,4 +388,89 @@ void canGetRatingsFromProductTest() throws Exception {
         verify(productService, times(1)).getAllRatingsFromProduct(dummyProduct.getId());
         verify(starRatingMapper, times(1)).mapToStarRatingSet(ratings);
     }
+
+    @Test
+    @WithMockUser
+    void canSubmitReviewTest() throws Exception {
+        // given
+        Product dummyProduct = createNonArchivedProduct();
+        Set<Review> reviewSet = new HashSet<>();
```

Comment:
provide a createReviewSet method in ReviewUtils, OR add reviews to the dummyProduct?

Reply:
moved to getReviewSetWithDummyReview() in ReviewUtils